lessons/day_02/java/src/main/resources/application.properties

```
@@ -4,4 +4,7 @@ logging.level.org.hibernate=ERROR
 spring.jpa.properties.hibernate.enable_lazy_load_no_trans=true
 
 spring.h2.console.enabled=true
+spring.datasource.url=jdbc:h2:mem:graphQL
 spring.datasource.name=graphQL
+
+spring.graphql.graphiql.enabled=true
```

Habría que agregar el path de graphiql para ser consistentes con el día 1
```suggestion
spring.graphql.graphiql.enabled=true
spring.graphql.graphiql.path=/playground
```